### PR TITLE
[close #168] Remove useless fields in changefeed

### LINKED ISSUE #168: cdc: Remove useless fields in changefeed
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Remove useless fields in changefeed (including `case-sensitive`, `force-replication`, `filter`, `mounter`, `cyclic-replication`, `consistent`), which will make users confused.

```
tikv-cdc cli changefeed query -c rawkv-replication-task --pd=http://10.2.100.122:2379
```
```
{
  "info": {
    "sink-uri": "tikv://10.2.100.61:2379/",
    "opts": {},
    "create-time": "2022-07-20T17:25:11.195983233+08:00",
    "start-ts": 434715783633567745,
    "target-ts": 0,
    "admin-job-type": 0,
    "sort-engine": "unified",
    "sort-dir": "",
    "config": {
      "case-sensitive": true,
      "enable-old-value": true,
      "force-replicate": false,
      "check-gc-safe-point": true,
      "filter": {
        "rules": [
          "*.*"
        ],
        "ignore-txn-start-ts": null
      },
      "mounter": {
        "worker-num": 16
      },
      "sink": {
        "dispatchers": null,
        "protocol": "",
        "column-selectors": null
      },
      "cyclic-replication": {
        "enable": false,
        "replica-id": 0,
        "filter-replica-ids": null,
        "id-buckets": 0,
        "sync-ddl": false
      },
      "scheduler": {
        "type": "keyspan-number",
        "polling-time": -1
      },
      "consistent": {
        "level": "none",
        "max-log-size": 64,
        "flush-interval": 1000,
        "storage": ""
      }
    },
    "state": "normal",
    "history": null,
    "error": null,
    "sync-point-enabled": false,
    "sync-point-interval": 600000000000,
    "creator-version": "v5.4.0-master"
  },
  "status": {
    "resolved-ts": 434731763970867404,
    "checkpoint-ts": 434731763708985535,
    "admin-job-type": 0
  },
  "count": 0,
  "task-status": [
    {
      "capture-id": "07684765-52df-42a0-8dd1-a4e9084bb7c1",
      "status": {
        "keyspans": {
          "15137828009456710810": {
            "start-ts": 434715783633567745,
            "Start": "cg==",
            "End": "cw=="
          }
        },
        "operation": {},
        "admin-job-type": 0
      }
    },
    {
      "capture-id": "aea1445b-c065-4dc5-be53-a445261f7fc2",
      "status": {
        "keyspans": null,
        "operation": null,
        "admin-job-type": 0
      }
    },
    {
      "capture-id": "f29496df-f6b4-4c1e-bfa3-41a058ce2144",
      "status": {
        "keyspans": null,
        "operation": null,
        "admin-job-type": 0
      }
    }
  ]
}
```

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu <yuping@pingcap.com>

### What problem does this PR solve?

Issue Number: close #168

Problem Description: Remove useless fields in changefeed

### What is changed and how does it work?
After this PR, result of `changefeed query` will be:
```
tikv-cdc cli changefeed query -c feed01
```
```
{
  "info": {
    "sink-uri": "tikv://127.0.0.1:32379",
    "opts": {},
    "create-time": "2022-07-21T17:10:25.056333448+08:00",
    "start-ts": 434738200588058625,
    "target-ts": 0,
    "admin-job-type": 0,
    "sort-engine": "memory",
    "sort-dir": "",
    "config": {
      "enable-old-value": false,
      "check-gc-safe-point": true,
      "sink": {
        "dispatchers": null,
        "protocol": "",
        "column-selectors": null
      },
      "scheduler": {
        "type": "keyspan-number",
        "polling-time": -1
      }
    },
    "state": "normal",
    "history": null,
    "error": null,
    "creator-version": "v1.0.0-master-dirty"
  },
  "status": {
    "resolved-ts": 434738200588058625,
    "checkpoint-ts": 434738200588058625,
    "admin-job-type": 0
  },
  "count": 0,
  "task-status": [
    {
      "capture-id": "947c00d7-8886-44af-b391-a9af85d0c5ab",
      "status": {
        "keyspans": {
          "15137828009456710810": {
            "start-ts": 434738200588058625,
            "Start": "cg==",
            "End": "cw=="
          }
        },
        "operation": {},
        "admin-job-type": 0
      }
    }
  ]
}
```

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Manual test (add detailed scripts or steps below)

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: cdc/Makefile [hunk 1]
```diff
@@ -43,8 +43,8 @@ FAILPOINT_DISABLE := $$(echo $(FAILPOINT_DIR) | xargs $(FAILPOINT) disable >/dev
 
 RELEASE_VERSION =
 ifeq ($(RELEASE_VERSION),)
-	RELEASE_VERSION := v5.4.0-master
-	release_version_regex := ^v5\..*$$
+	RELEASE_VERSION := v1.0.0-master
```
Comment (haojinming) [inline cdc/Makefile:46]: We may need check the tikv version and apiversion to make sure the compatibilcity.

**Review by haojinming — APPROVED** [review]

File: cdc/Makefile (hunk 1 above)
Comment (pingyu) [reply cdc/Makefile:46]: Agree. I open an issue https://github.com/tikv/migration/issues/170 for this.

File: cdc/cdc/model/changefeed_test.go [hunk 2]
```diff
@@ -119,45 +68,12 @@ func TestFillV1(t *testing.T) {
...
-					IgnoreDBs: []string{"test", "sys"},
-				},
-				IgnoreTxnStartTs: []uint64{1, 2},
-				DDLAllowlist:     []model.ActionType{1, 2},
-			},
-			Mounter: &config.MounterConfig{
-				WorkerNum: 64,
-			},
 			Sink: &config.SinkConfig{
 				DispatchRules: []*config.DispatchRule{
```
Comment (zeminzhou) [inline cdc/cdc/model/changefeed_test.go:72]: Also remove here?

**Review by zeminzhou — APPROVED** [review]

File: cdc/cdc/model/changefeed_test.go (hunk 2 above)
Comment (pingyu) [reply cdc/cdc/model/changefeed_test.go:72]: `DispatchRules` ? Would be useful when we support message queue as downstream. So I keep them.

**Pull request merged**